scripts/kind-with-registry.sh

```diff
@@ -168,7 +187,7 @@ function upload_to_blob() {
   local blob_name=$2
 
   echo "Uploading ${file_path} to '${AZWI_STORAGE_ACCOUNT}' storage account"
-  az storage blob upload \
+  az storage blob upload --auth-mode "key" \
```

Comment:
we should be able to use `login` here since we added role assignments for the storage accoutns to the azure cli sp

Reply:
But we need to do it based on the Azure cli login in method

Reply:
Updated, please take a look